### PR TITLE
Added overlay

### PR DESCRIPTION
**Pull request by JLambertazzo**
Issue: #83 Add overlay when sidebar opens on small devices
I added a div as an overlay to darken everything but the sidebar on smaller devices.

### CODE REVIEW

File: assets/css/sidebar.css [hunk 1]
```diff
@@ -97,4 +102,13 @@
...
     z-index: 100;
   }
+  
+  #shade.shown {
+    width: 100vw;
+    height: 100vh;
+    z-index: 9;
+    position: fixed;
+    background-color: black;
+    opacity: 30%;
```
Comment (shahednasser) [inline assets/css/sidebar.css:112]: Change opacity to 50%

File: assets/css/sidebar.css [hunk 2]
```diff
@@ -57,6 +57,11 @@
   color: #fff;
 }
 
+#shade {
+  opacity: 0%;
+  transition: opacity 2s;
```
Comment (shahednasser) [inline assets/css/sidebar.css:62]: change transition duration to 0.2s instead of 2

**Review by shahednasser — CHANGES_REQUESTED** [review]
It all works perfectly! However, we just added a new feature where clicking outside the sidebar can closes it. Can you update the code, and toggle the class "shown" in that event as well? If you need any assistance in doing this please let me know.

**Comment by JLambertazzo** [comment]
Done!

**Review by shahednasser — CHANGES_REQUESTED** [review]
It all works perfectly! But I just noticed that when clicking on one of the links in the sidebar, the overlay is still there so it needs to be added to that event. I think we need to make closing the sidebar a function itself so that we don't have so much repeated code. For now, please add the toggle to that event as well. If you find other events in the file that closes the sidebar as well, please add it there.

**Comment by JLambertazzo** [comment]
Oops, fixed that!

**Comment by shahednasser** [comment]
Amazing work! Thank you.

@all-contributors please add @JLambertazzo for code

**Pull request merged**